deploy/migrate-to-keycloak7.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
```

Comment:
we've to have a check for mandatory env vars. Else the script will fail midst some operation.

Reply:
Thats what set -u does right?

Reply:
:+1: